src/lowering/desugar.ml

```diff
@@ -169,6 +169,9 @@ and exp' at note = function
   | S.CallE ({it=S.AnnotE ({it=S.PrimE "deadline";_},_);_}, _, {it=S.TupE es;_}) ->
     assert (es = []);
     I.PrimE (I.ICReplyDeadlinePrim, [])
+  | S.CallE ({it=S.AnnotE ({it=S.PrimE "canister_subnet";_},_);_}, _, {it=S.TupE es;_}) ->
+    assert (es = []);
+    I.PrimE (I.OtherPrim "canister_subnet", [])
```

Comment:
This is not covered in line 202, we could think about introducing a non-`OtherPrim` one.

Reply:
ah, maybe it is covered in line 200?

Reply:
35e0c404aa48b40958095584e26b7ab884fb026a